ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml

```diff
@@ -76,7 +76,7 @@ StatusModal {
                                 hoverEnabled: true
                                 anchors.fill: parent
                                 cursorShape: Qt.PointingHandCursor
-                                onClicked: communitySectionModule.acceptRequestToJoinCommunity(id)
+                                onClicked: communitySectionModule.acceptRequestToJoinCommunity(id, popup.communityData.id)
```

Comment:
This should go thru the store (using the method below), not directly via the module

Reply:
fixed